nixos/modules/tasks/network-interfaces.nix

```
@@ -975,6 +975,14 @@ in
         message = ''
           Temporary addresses are only needed when IPv6 is enabled.
         '';
+      })) ++ (forEach interfaces (i: {
+        assertion = cfg.useDHCP -> !builtins.hasAttr i.name cfg.bridges;
+        message = ''
+	  ${i.name} is a bridge, while `networking.useDHCP` is enabled.
+	  Dhcpcd doesn't give IPv4 addresses to bridges by default anymore, so you have
```

```suggestion
	  dhcpcd doesn't give IPv4 addresses to bridges by default anymore, so you have
```

just a nitpick, but I don't see it capitalized frequently.

---

Right, I changed that and rebased to latest `release-20.03`.